src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/entities/AutoBreeder.java

```diff
@@ -111,7 +111,7 @@ protected void tick(Block b) {
 
         for (Entity n : b.getWorld().getNearbyEntities(b.getLocation(), 4.0, 2.0, 4.0, this::canBreed)) {
             for (int slot : getInputSlots()) {
-                if (SlimefunUtils.isItemSimilar(inv.getItemInSlot(slot), organicFood, false)) {
+                if (SlimefunUtils.compareItem(inv.getItemInSlot(slot), organicFood)) {
```

Comment:
This compares with an unobtainable blank organic food item.
So this would break this machine.